cmd/osv-scanner/main_test.go

```diff
@@ -309,6 +310,100 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestRun_StdIn(t *testing.T) {
+	t.Parallel()
+
+	tests := []cliTestCase{
+		{
+			name:         "1 PURL from stdin",
+			args:         []string{"", "--experimental-stdin-purl"},
+			stdin:        "pkg:npm/ansi-html@0.0.1\n",
+			wantExitCode: 1,
+			wantStdout: `
+            Enter Package URLs, one per line, finishing with EOF:
```

Comment:
nit: it would be good to keep the indentation consistent for these (it should be intended two spaces in from the start of `wantStdout`)